# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1539932112 14400
#      Fri Oct 19 02:55:12 2018 -0400
# Node ID a0a8d4dd41e770af3a9e0c9d3182cfc0ab63e7f7
# Parent  ceb508ea92ee5504fd6436405106e978ea490f6e
remove debugging code accidentally left in previous change

* octave-qt-link.cc (octave_qt_link::do_file_remove):
Remove debugging code accidentally left in previous change.

diff --git a/libgui/src/octave-qt-link.cc b/libgui/src/octave-qt-link.cc
--- a/libgui/src/octave-qt-link.cc
+++ b/libgui/src/octave-qt-link.cc
@@ -404,17 +404,16 @@ namespace octave
   void octave_qt_link::do_change_directory (const std::string& dir)
   {
     emit change_directory_signal (QString::fromStdString (dir));
   }
 
   void octave_qt_link::do_file_remove (const std::string& old_name,
                                        const std::string& new_name)
   {
-    return;
     // Lock the mutex before signaling
     lock ();
 
     // Emit the signal for the editor for closing the file if it is open
     emit file_remove_signal (QString::fromStdString (old_name),
                              QString::fromStdString (new_name));
 
     // Wait for the GUI and unlock when resumed
